src/components/localization/language-dropdown.tsx

```diff
@@ -100,10 +96,9 @@ const FadeOutUp = keyframes`
 
 const ItemContainer = styled.div<{ is_open: boolean }>`
     background-color: var(--color-white);
-    padding: 1.6rem 0.8rem;
+    padding: 1.6rem 0.8rem 0rem 0rem;
```

Comment:
No unit is needed for length 0.

```suggestion
    padding: 1.6rem 0.8rem 0 0;
```

Reply:
ok